drivers/mspi/mspi_dw.c

```diff
@@ -1231,6 +1235,20 @@ static int _api_xip_config(const struct device *dev,
 	return 0;
 }
 
+static int api_timing_config(const struct device *dev,
```

Comment:
I believe @danieldegrasse you added this behind `CONFIG_MSPI_XIP` when it was not intended to be such.